egs/lre07/v2/run.sh

```diff
@@ -0,0 +1,234 @@
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 12 --cmd "$train_cmd" \
+  data/train exp/make_mfcc $mfccdir
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 12 --cmd "$train_cmd" \
+  data/lre07 exp/make_mfcc $mfccdir
+
+# Extract DNN features
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc_hires.conf --nj 12 --cmd "$train_cmd" \
+    data/train_dnn exp/make_mfcc $mfccdir
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc_hires.conf --nj 12 --cmd "$train_cmd" \
+    data/lre07_dnn exp/make_mfcc $mfccdir
+
+for name in lre07_dnn train_dnn lre07 train; do
+    utils/fix_data_dir.sh data/${name}
+done
+
+lid/compute_vad_decision.sh --nj 12 --cmd "$train_cmd" data/train \
+  exp/make_vad $vaddir
+lid/compute_vad_decision.sh --nj 12 --cmd "$train_cmd" data/lre07 \
+  exp/make_vad $vaddir
+
+
+for name in train lre07; do
+  cp data/${name}/vad.scp data/${name}_dnn/vad.scp
+  cp data/${name}/utt2spk data/${name}_dnn/utt2spk
+  cp data/${name}/spk2utt data/${name}_dnn/spk2utt
+  utils/fix_data_dir.sh data/${name}
+  utils/fix_data_dir.sh data/${name}_dnn
+done
+
+# Subset training data for faster sup-GMM initialization.
+utils/subset_data_dir.sh data/train 32000 data/train_32k
+utils/fix_data_dir.sh data/train_32k
+utils/subset_data_dir.sh data/train_dnn 32000 data/train_dnn_32k
+utils/fix_data_dir.sh data/train_dnn_32k
+
+# Initialize a full GMM from the DNN posteriors and language recognition
+# features. This can be used both alone, as a UBM, or to initialize the
+# i-vector extractor in a DNN-based system.
+lid/init_full_ubm_from_dnn.sh --nj 40 --cmd "$train_cmd -l mem_free=6G,ram_free=6G" \
+  data/train_32k \
+  data/train_dnn_32k $nnet exp/full_ubm
+
+
+# Train an i-vector extractor based on the DNN-UBM.
+lid/train_ivector_extractor_dnn.sh --cmd "$train_cmd -l mem_free=80G,ram_free=80G" \
+				   --min-post 0.015 \
+				   --ivector-dim 600 \
+				   --num-iters 5 exp/full_ubm/final.ubm $nnet \
+				   data/train \
+				   data/train_dnn \
+				   exp/extractor_dnn
+
+# Filter out the languages we don't need for the closed-set eval
+cp -r data/train data/train_lr
+utils/filter_scp.pl -f 2 $languages <(lid/remove_dialect.pl data/train/utt2lang) \
+  > data/train_lr/utt2lang
+utils/fix_data_dir.sh data/train_lr
+
+echo "**Language count for logistic regression training (after splitting long utterances):**"
+awk '{print $2}' data/train_lr/utt2lang | sort | uniq -c | sort -nr
+
+cp -r data/train_dnn data/train_lr_dnn
+utils/filter_scp.pl -f 2 $languages <(lid/remove_dialect.pl data/train_dnn/utt2lang) \
+  > data/train_lr_dnn/utt2lang
+utils/fix_data_dir.sh data/train_lr_dnn
+
+echo "**Language count for logistic regression training (after splitting long utterances):**"
+awk '{print $2}' data/train_lr_dnn/utt2lang | sort | uniq -c | sort -nr
+
+# Extract i-vectors using the extractor with the DNN-UBM
+lid/extract_ivectors_dnn.sh --cmd "$train_cmd -l mem_free=10G,ram_free=10G" --nj 40 \
+			    exp/extractor_dnn \
```

Comment:
we normally use 2 spaces for indent, not 16 or whatever you are using here.  same issue exists elsewhere.